pkg/datapath/linux/ipsec/ipsec_linux_test.go

```diff
@@ -1,6 +1,8 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright Authors of Cilium
 
+//go:build unparallel
```

Comment:
[again newbee question]
What is the part in the tests that causes problem? I'm guessing `UpsertIPsecEndpoint` being called from different parallel tests? Asking to make sure.

Reply:
Usual errors that were happening in the past: https://github.com/cilium/cilium/pull/32954
https://github.com/cilium/cilium/issues/32902

so yes, cleaning up ipsec state from other parallel packages.

Reply:
Thanks for the context :+1: